RoutesApp_ios/repository/models/remote/Line.swift

```diff
@@ -24,6 +24,23 @@ struct LineRoute: Codable, Equatable {
     let blackIcon: String
     let whiteIcon: String
     let color: String
+    static func getWalkLineRoute(routePoints: [Coordinate]) -> LineRoute {
+        return LineRoute(
+            name: "Walk",
```

Comment:
You can use the static variable instead of the string

Reply:
call the variable instead of adding the Walk string manually if its supposed to be the same